docs/source/gui.rst

```diff
@@ -6,16 +6,213 @@ Graphical User Interface
 
 The Waveform Editor provides a Graphical User Interface (GUI), with which you can:
 
-* Load an existing YAML file from disk
-* Create/remove groups
-* Create/remove waveforms
-* View or edit waveforms
-* Store changes back to YAML
+* Create, load, edit, and save waveform configurations from YAML files.
+* View plots of one or multiple waveforms simultaneously for comparison.
+* Edit waveforms through a text editor with live plot updates and error
+  highlighting.
+* Export waveform data to various formats, including IDS, CSV, PNG, and PCSSP XML.
+* Graphically edit and design the desired plasma shape using NICE.
 
-The GUI can be launched using:
+Launching the GUI
+-----------------
+
+The GUI can be launched using the following command in your terminal:
```

Comment:
Depending on the user's configuration, it could be that the webbrowser doesn't launch automatically. Can you add something here like:

- This should open a webbrowser
- If not, the command will print the web address (localhost:12345) where the app is running. You can open a webbrowser and go there yourself